### ISSUE TITLE
get sys info errors

### ISSUE THREAD

**Issue by proftsw** [DaveGut/SmartThings_Cloud-Based_TP-Link-Plugs-Switches-Bulbs#14]
[tp link smartthings errors.txt](https://github.com/DaveGut/SmartThings_Cloud-Based_TP-Link-Plugs-Switches-Bulbs/files/2368290/tp.link.smartthings.errors.txt)

![screenshot_20180910-142009_smartthings 2175](https://user-images.githubusercontent.com/34167427/45323070-4f061f80-b507-11e8-9212-f6e6afa2b5a5.jpg)

Hi Dave.  First of all thank you for the TP Link integration.  I have been using it with just a single HS100 plug for some time now without issue.  I have just recently added the HS110, HS210 and LB120 devices to my Kasa account and added the appropriate device handlers for the cloud based integration.  However, I immediately get a security access denied error in my live logging and a get_sys info error for the LB120.  The smartthings buttons are also stuck on green 'waiting' even when the refresh is set to the minimum timeframe and never updates.  Attached is my live logging text and a screenshot of the 'waiting' delay.  Do you have any insight for me on how to correct this?  Thanks again.

**Comment by DaveGut** [comment]
The get_sys error is now corrected (I was out of pocket for a while and TP-Link changed the format of the return when a bulb is turned on/off.  It messed up my parsing logic and caused an error reading the data.

You will need to reload the BULB driver.  As far as security error - ?????????

Dave

**Comment by proftsw** [comment]
Works perfectly now.  Thanks for the update.

**Comment by proftsw** [comment]
Awesome.  I confirmed that everything (bulbs, switches, plugs) works perfectly now.  Thank you for your help.

Sorry about the ‘security’ listing.  I was not implying anything malicious.  It was just a word that caught my eye in the live logging text.  ----- e23b879a-fe02-48ad-ac0d-00c71fb2b046  2:14:14 PM: error org.springframework.security.access.AccessDeniedException: Access is denied @line -1 (doCall)
I have since edited the title of my ‘issue’ in GitHub and have closed it.  This was the ONLY time I had seen this access is denied error.  None since.

I’m grateful for all your work on this.

At least now I know I can replace all of my old, annoying GE Link bulbs with the TP Link line…
